nilearn/datasets/func.py

```diff
@@ -902,6 +898,24 @@ def fetch_localizer_contrasts(
     return Bunch(ext_vars=csv_data, description=fdescr, **files)
 
 
+def _check_inputs_fetch_localizer_contrasts(contrasts):
+    if isinstance(contrasts, str):
+        raise ValueError(
+            "Contrasts should be a list of strings, but "
+            f'a single string was given: "{contrasts}"'
+        )
+    unknown_contrasts = [
+        x
+        for x in contrasts
+        if (x not in ALLOWED_CONTRASTS and x not in CONTRAST_NAME_WRAPPER)
+    ]
+    if unknown_contrasts:
+        raise ValueError(
```

Comment:
Maybe a couple of tests for the ValueErrors here? Or would that be out-of-scope for the PR?